scripts/capture_tui_assets.py

```
@@ -0,0 +1,506 @@
+
+
+def build_gif(frames: list[tuple[str, int]], out_path: Path) -> None:
+    """Render SVG frames with Chromium and assemble an animated GIF."""
+    from PIL import Image
+    from playwright.sync_api import sync_playwright
+
+    pngs: list[bytes] = []
+    with sync_playwright() as p:
+        try:
+            browser = p.chromium.launch()
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)